modules/search/js/search-widget-filters.js

```diff
@@ -0,0 +1,14 @@
+jQuery( document ).ready( function() {
```

Comment:
Typically with the document ready callback we accept the jQuery object as an arg called '$', like this:

```js
jQuery( document ).ready( function( $ ) { 
  //...
  anchor = $( this ).closest( 'a' );
  //...
}
```

Reply:
It's a bit different I think. `jQuery` is a global, and we pass it in to our script via an immediately invoked anonymous function (this is always a mouthful).

```js
( function( $ ) { 
    $( document ).ready( fn );
} ) ( jQuery ) ;
```

Reply:
I think both will work, based on a quick reading of the jQuery docs. Which we prefer is up to you. I see both styles in Jetpack - JITMs and Comment Like Counts use my style above, but IDC notices and the Jetpack Connection Banner use @ebinnion's style. I am going to approve this - it works anyway, and I don't see a reason to hold it up for this stylistic reason when there isn't a clear standard in the code.